packages/microcosm/test/unit/domain/register.test.js

```diff
@@ -63,4 +63,38 @@ describe('Domain::register', function() {
 
     expect(repo.state.test).toBe('test')
   })
+
+  it('should have state before an action resolves', async () => {
+    expect.assertions(2)
```

Comment:
what does this line do?

Reply:
Fails the test if 2 assertions aren't given. This is useful for making sure code in `try/catch` executes, or that callbacks don't duplicatively fire (particularly nice for catching cleanup issues)

Reply:
neat! thanks for the explanation!